src/interrupt.c

```diff
@@ -569,11 +570,29 @@ void set_AI_interrupt(int active)
 /**
  * @brief Enable or disable the VI interrupt
  *
+ * The VI interrupt is generated when the VI begins displaying a specific line
+ * of the display output. The line number configured always refers to the
+ * final TV output, so it should be either in the range 0..479 (NTSC) or
+ * 0..575 (PAL).
```

Comment:
```suggestion
 * final TV output, so it should be either in the range 0..523 (NTSC) or
 * 0..623 (PAL).
```

I got them wrong the first time, these are the correct ranges (including vblank).

Reply:
I'm having a hard time finding a neat primary source but I've seen in various places (e.g. https://www.sony.com/electronics/support/articles/00006681 )
that there are 525 (NTSC) and 625 (PAL) lines
0-indexed that would be 0..524 and 0..624 ?

(if you have a proper spec of NTSC/PAL handy I'd like to see it)

Reply:
Yes, it should be 0..524 and 0..624. I tried to poll the VI current line register in my PAL console, and the highest value I got was 624 like you would expect.

Reply:
Fixed